HarpGuru/Controls/CovariantsGroupList/getCovariantsGroupList/index.test.ts

```diff
@@ -0,0 +1,107 @@
+import { PitchIds, PozitionIds } from 'harpstrata'
+import type { CovariantsGroup } from 'harpstrata'
+
+import { ControlVarsPrimer, CovarianceParts } from '../types'
+
+import { getCovariantsGroupList } from './index'
+
+test('getCovariantsGroupList returns a complete mapping of the control covariants to the dependent covariant', () => {
+  const controlVarsPrimer: ControlVarsPrimer = {
```

Comment:
I don't think this matters very much. It would be a nice place to have some consistency, just like everywhere else, but I think it's a lot of auditing for not a lot of value. It's probably good to do both since it changes the intention of the test. If the function is close to the surface of the module and not being tested with all the variations of it's input, then it makes sense to test it's parent type input. If we're wanting to do a complete test of all of the possible subtypes input then it makes sense to identify which types they are.